Assignment 2/180501040/attendanceStudents.py

```diff
@@ -0,0 +1,11 @@
+l=[]
+x=int(input("Enter no of student"))
+print("Enter your string")
+for i in range(0,x):
+    a=str(input())
+    l.append(a)
+print("The student present were:")
+for j in l:
+    z="P" in j

```

Comment:
There's a flaw in your logic here. Checking if 'P' is in the string will lead to marking people with 'P' in their names present, even when they are absent. Think of the following test case:

`Priya-A`

Your code will mark Priya as present even when absent. Correct the code, commit the changes and comment acknowledging the same.